### PR TITLE
Freeze Queue

### PR DESCRIPTION
**Pull request by ThrowsException**
#214 

Makes the program not selectable from the participant info form if the program has been marked as frozen from the queue

### CODE REVIEW

File: frontend/src/components/AllQueues.js [hunk 1]
```diff
@@ -128,6 +142,17 @@ const AllQueues = observer(() => {
         </Tabs>
       </AppBar>
       <QueueTable queueData={tabValue + 1} />
+
+      <div
+        style={{
```
Comment (MikeyManoguerra) [inline frontend/src/components/AllQueues.js]: best to avoid inline styles, can you create a scss/queue file and add this there

**Comment by ThrowsException** [comment]
@MikeyManoguerra thanks for the review. I revamped this a little bit. Instead of just blindly flipping the state when its clicked the button will wait for a valid server response. I also went a fixed the AllQueue.test.js file since it was failing due to some other PR's. Also added a test for the button. Probably want to give this a once over again.

**Comment by MikeyManoguerra** [comment]
Looks good!
Its ok to merge, i believe convention still is once you get approved you merge your own changes.

Some things outside the scope of this ticket we'll need to think about: 
- the difference between "frozen" and "closed". 
- A daily cycle : does "is_ frozen" reset to false every day?  This can fold into some sort of daily reset routine, That should be discussed in full because there are definitely other undecided lifecycle type things to consider

**Comment by ThrowsException** [comment]
@MikeyManoguerra Yea I noticed the is closed. if its decided later to use that instead it should be easy to switch over. ill get the branch updated and merged.

**Comment by ThrowsException** [comment]
I guess by rebasing it invalidates the review. Sorry mind approving again and ill get this merged.

**Comment by MikeyManoguerra** [comment]
@ThrowsException they're moving too fast! message me on slack and ill see this quicker

**Pull request merged**